content/en/workflows/setup.md

```diff
@@ -1,9 +1,82 @@
 ---
-title: Set up Workflows
+title: Authentication
 kind: documentation
 disable_toc: false
 further_reading:
-- link: "logs/processing/pipelines"
+- link: "/integrations/"
   tag: "Documentation"
-  text: "Log processing pipelines"
----
\ No newline at end of file
+  text: "Learn about integrations"
+---
+
+<div class="alert alert-warning">Workflows are in public beta. If you have any feedback or questions, contact <a href="/help">Datadog support</a>.</div>
+
+Because workflow actions connect with external software systems, you may need to authenticate your Datadog account to the corresponding integration or integrations. A workflow can run successfully only if every workflow action that requires authentication can verify the identity of your Datadog account.
+
+Integration actions can be authenticated in two ways:
+- Credentials and permissions configured in the integration tile
+- Custom connection credentials
```

Comment:
Perhaps we could add some information here on why a user might create a custom connection instead of using an integration tile. If I recall correctly, some generic actions don't have integrations tiles and must be connected this way. But there might be other reasons, too.

Reply:
Yes, good point. I'll add some info in the section below about custom connection credentials.